src/components/views/HomeView/HomeView.tsx

```diff
@@ -14,7 +16,7 @@ export const HomeView = () => {
   }, []);
   return (
     <BackgroundImage>
-      <ScreenTitle>Kampanie</ScreenTitle>
+      <ScreenTitle>{t('Campains')}</ScreenTitle>
```

Comment:
zrób se tak, dodaj enum z kluczami translacji, takie stringi z palca to gówno

Zrób sobie custom hook'a zamiast useTranslation z libki co ci zaakceptuje tylko enum'a 

Coś takiego

`enum TranslationKeys {
DUPA = 'DUPA'
}

export const useTranslationInternal = () => {
const {t} = useTranslation()

return useMemo(() => ({
t(key: TranslationKeys, options?: TOptions) {return t(key, options}
}), [])
}
// I bez hook'a

export const translate = (key: TranslationKeys, options?: TOptions) => translate(key,options)
`